mesh/pmesh.cpp

```diff
@@ -2436,8 +2439,7 @@ GetSharedFaceTransformations(int sf, bool fill2)
    // for ghost faces we need a special version of GetFaceTransformation
    if (is_ghost)
    {
-      FaceElemTr.Face =
-         GetGhostFaceTransformation(&FaceElemTr, face_type, face_geom);
+      GetGhostFaceTransformation(&FaceElemTr, face_type, face_geom);
```

Comment:
This change introduced a bug -- `GetGhostFaceTransformation` needs to setup `FaceElemTr` itself as the face transformation, not the internal variable `FaceTransformation`.

Reply:
I don't think I've worked with ghost elements in MFEM before.  If you can point me to an example of creating a simple test mesh with ghost elements I'll try to add a unit test for this and resolve the bug.

Reply:
We can put the fix in PR #1529

Reply:
Okay, I'll push the fix there.

Ghost face elements are present in a mesh when a non-conforming face is between two different neighbor processors.